webrender_api/src/image.rs

```diff
@@ -222,44 +298,64 @@ pub trait BlobImageHandler: Send {
 
 /// A group of rasterization requests to execute synchronously on the scene builder thread.
 pub trait AsyncBlobImageRasterizer : Send {
+    /// Rasterize the requests.
     fn rasterize(&mut self, requests: &[BlobImageParams]) -> Vec<(BlobImageRequest, BlobImageResult)>;
 }
 
 
+/// FIXME: document
 #[derive(Copy, Clone, Debug)]
 pub struct BlobImageParams {
+    /// FIXME: document
     pub request: BlobImageRequest,
+    /// FIXME: document
     pub descriptor: BlobImageDescriptor,
+    /// FIXME: document
```

Comment:
/// An optional sub-rectangle of the image to avoid re-rasterizing the entire image when only a portion is updated.
///
/// If set to `None` the entire image is rasterized.